yamllint/rules/key_duplicates.py

```diff
@@ -16,6 +16,18 @@
 """
 Use this rule to prevent multiple entries with the same key in mappings.
 
+.. rubric:: Options
+
+* ``forbid-duplicated-merge-keys`` to forbid the usage of multiple merge keys.
```

Comment:
For consistency with other rules, I would propose:
```suggestion
* Use ``forbid-duplicated-merge-keys`` to forbid the usage of multiple merge
  keys.
```
or:
```suggestion
* Set ``forbid-duplicated-merge-keys`` to ``true`` to forbid the usage of
  multiple merge keys.
```

And also `merge keys.` → `merge keys ``<<``.`?

Reply:
Went with the first option.